grakn-factory/titan-factory/src/main/java/ai/grakn/graph/internal/GraknTitanGraph.java

```diff
@@ -73,16 +72,6 @@ public void openTransaction(GraknTxType txType){
     }
 
     @Override
-    public void closeSession(){
-        super.closeSession();
-
-        //Close the system graph if possible
-        if(!getKeyspace().equalsIgnoreCase(SystemKeyspace.SYSTEM_GRAPH_NAME)) {
```

Comment:
YES! I did not even think about this but this is another wonderful benefit of making the system keyspace an engine concern